pkg/test/expectations/expectations.go

```diff
@@ -306,6 +306,39 @@ func ExpectProvisioned(ctx context.Context, c client.Client, cluster *state.Clus
 	return bindings
 }
 
+//nolint:gocyclo
+func ExpectProvisionedNoNode(ctx context.Context, c client.Client, cluster *state.Cluster, cloudProvider cloudprovider.CloudProvider, provisioner *provisioning.Provisioner, pods ...*corev1.Pod) ([]*v1.NodeClaim, error) {
```

Comment:
Since this function is a subset of the other function, can you just use this function in ExepctProvisionedNoBinding()?